libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1687,3 +1687,57 @@ export type Type3 = MessagingExtensionResultType;
  * @enum {string}
  */
 export type Action = 'accept' | 'decline';
+
+/**
+ * @interface
+ * Specific details of a Teams meeting.
+ */
+export interface MeetingDetails {
+    /**
+     * @member {string} [id] The meeting's Id, encoded as a BASE64 string.
+     */
+    id: string;
+    /**
+     * @member {string} [msGraphResourceId] The MsGraphResourceId, used specifically for MS Graph API calls.
+     */
+    msGraphResourceId: string;
+    /**
+     * @member {Date} [scheduledStartTime] The meeting's scheduled start time, in UTC.
+     */
+    scheduledStartTime: Date;
+    /**
+     * @member {Date} [scheduledEndTime] The meeting's scheduled end time, in UTC.
+     */
+    scheduledEndTime: Date;
```

Comment:
Like [.NET](https://github.com/microsoft/botbuilder-dotnet/pull/5655#discussion_r649444421), I'm open to changing these to `string`.

Reply:
Does `msrest` parse JSON strings into a `Date` object?

Reply:
![image](https://user-images.githubusercontent.com/40401643/121926863-40ba4300-ccf3-11eb-9eab-f18ee327907c.png)

Sure does. (works for .NET, too)